syntax = "proto2";

import "lib/common.proto";

message Commodore64DecoderProto {}

message Commodore64EncoderProto {
	optional double post_index_gap_us = 1 [default=0.0,
		(help) = "post-index gap before first sector header."];
	optional double clock_compensation_factor = 2 [default=1.0,
		(help) = "scale the output clock by this much."];
	optional bool forty_track_drive = 3 [default=false, (help) = "is a 40 track drive used."];
}

